cmd/osv-scanner/main_test.go

```diff
@@ -508,8 +508,19 @@ func TestRun_LocalDatabases(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
 
-			testDir, cleanupTestDir := createTestDir(t)
-			defer cleanupTestDir()
+			var testDir string
+			var cleanupTestDir func()
+
+			if os.Getenv("TEST_ACCEPTANCE") != "true" {
+				testDir = filepath.Join(os.TempDir(), "osv-scanner-test-0")
```

Comment:
couldn't we just do this outside of the test loop? that should let you reuse `createTestDir` and not need to explicitly name a directory for the normalizer.

i.e. basically do this out of the loop, then here check for `testDir === ""` and if so `testDir, cleanupTestDir = createTestDir(t); defer cleanupTestDir()`